### DISCUSSION TITLE
How could recipients delete reports?

### DISCUSSION THREAD

**Discussion by creativeanalyst** [globaleaks/globaleaks-whistleblowing-software#4548] [Q&A] [answered]
Advance settings - Allow recipients to delete reports seems have some issues

Seems this setting is not working. 
I have left this option unchecked to not allow recipient to delete any report.

Noticed system allows recipient user to delete reports. Noticed this behavior in our try demo site.

**Comment by evilaliv3** [comment, marked as answer]
@creativeanalyst: Users can be enabled to delete reports at two levels:
- locally setting the privilege on the specific user
- globally setting the privilege in the advanced settings.

Would you please check that on your system both the settings are disabled?

**Comment by creativeanalyst** [comment]
Thank you pointing me to the right place to make changes! 
I checked and now it seems working as expected after I changed the settings for recipient user. 
Initially I had global settings only. I think this can be closed now.
